### ISSUE TITLE
Possible missing carriage return in 2018 J Playground Port

### ISSUE THREAD

**Issue by joebo** [jsoftware/j-playground#20]
In the modified websocket.js

```
send: function(cmd) { 
    //console.log(cmd);
    var ret = jdo1(cmd);
    //tcmreturn slices the first character off
    tcmreturn(' ' + ret);
  }
  ```
  example
  
  ```
  ]a=: i.5 
0 1 2 3 4a
```

_Originally posted by @jonghough in https://github.com/jsoftware/j-playground/issues/19#issuecomment-1091703342_

**Comment by joebo** [comment]
@jonghough  I can't reproduce this issue. Can you provide the specific steps you do to reproduce?

My steps:
1. Copy paste the following into the terminal: ]a=: i.5 
2. Hit enter

Looks fine

Attempt 2
1. Copy paste the following into the code area: ]a=: i.5 
2. Hit ctrl+enter to execute

Looks fine

Browser:

Google Chrome	96.0.4664.110 (Official Build) (64-bit) (cohort: Stable)
Revision	d5ef0e8214bc14c9b5bbf69a1515e431394c62a6-refs/branch-heads/4664@{#1283}
OS	Windows 10 Version 20H2 (Build 19042.1415)


![image](https://user-images.githubusercontent.com/151573/162340369-edf83ac6-89c0-408e-8a8c-27cb09055a25.png)

**Comment by joebo** [comment]
My bad - you already fixed this in 8f49aaca7e0874a137ac01fbf17ffdf31dc0fc9f. Thanks for the fix!
